OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -42,6 +45,9 @@ public class WithDecorationInfo : UpgradableTraitInfo, ITraitInfo
 		[Desc("Should this be visible to enemy players?")]
 		public readonly bool ShowToEnemies = false;
 
+		[Desc("Should this only be rendered when actor is in world?")]
+		public readonly bool OnlyWhenInWorld = true;
```

Comment:
I primarily had paradropped units with decorations in mind when writing this. They are not in the world, but still need rendering. How do you propose dealing with this?